qsirecon/config.py

```diff
@@ -567,83 +541,14 @@ def _process_value(value):
 class workflow(_Config):
     """Configure the particular execution graph of this workflow."""
 
-    anat_modality = None
-    """Modality to use as the anatomical reference. Images of this
-    contrast will be skull stripped and segmented for use in the
-    visual reports and reconstruction. If --infant, T2w is forced."""
-    anat_only = False
-    """Execute the anatomical postprocessing only."""
-    anatomical_template = None
-    """Keeps the :py:class:`~niworkflows.utils.spaces.SpatialReferences`
-    instance keeping standard and nonstandard spaces."""
-    b0_threshold = None
-    """Any value in the .bval file less than this will be considered a b=0 image."""
-    b0_motion_corr_to = None
-    """Perform SHORELine's initial b=0-based registration to first volume?
-    Or make a template? Either 'iterative' or 'first'"""
-    b0_to_t1w_transform = None
-    """Transformation model for intramodal registration."""
-    b1_biascorrect_stage = None
-    """The stage of processing at which to apply B1 bias correction. Either "final" (after
-    resampling), "none" (skipped entirely) or "legacy" (before concatenation)."""
-    cifti_output = None
```

Comment:
I could see a ScalarMapper writing to cifti sometime in the future

Reply:
I'm hoping that, by that time, we'll have CIFTIs defined in `--output-spaces` instead.